packages/expo-notifications/src/TokenEmitter.ts

```diff
@@ -0,0 +1,24 @@
+import { EventEmitter, Subscription, Platform } from '@unimodules/core';
+import { DevicePushToken } from './getDevicePushTokenAsync';
+import PushTokenManager from './PushTokenManager';
+
+export type TokenListener = (token: DevicePushToken) => void;
+
+// Web uses SyntheticEventEmitter
+const tokenEmitter = new EventEmitter(PushTokenManager);
+const newTokenEventName = 'onDevicePushToken';
+
+export function addTokenListener(listener: TokenListener): Subscription {
+  const wrappingListener = ({ devicePushToken }) =>
+    // @ts-ignore: TS can't decide what Platform.OS is.
+    listener({ data: devicePushToken, type: Platform.OS });
+  return tokenEmitter.addListener(newTokenEventName, wrappingListener);
+}
+
+export function removeTokenSubscription(subscription: Subscription) {
```

Comment:
removePushTokenSubscription
(although, is this function needed if developers can just call `subscription.remove()`?)